osf_tests/test_storage_usage_limits.py

```diff
@@ -22,7 +22,7 @@ def test_limit_default(self, node):
         assert node.storage_limit_status == StorageLimits.DEFAULT
 
     def test_storage_limits(self, node):
-        GBs = 1024 ** 3.0
+        GBs = 10 ** 9
```

Comment:
You are using GBs in some places and GiBs in others. Maybe we should put the GiB into `website.settings.defaults` and just import that into wherever you need to use it, so we have one place it's defined.